ddtrace/internal/datadog/profiling/dd_wrapper/src/exporter.cpp

```diff
@@ -472,12 +478,11 @@ Profile::flush_sample()
         .labels = { &labels[0], cur_label },
     };
 
-    ddog_prof_Profile_AddResult address = ddog_prof_Profile_add(ddog_profile, sample);
-    if (address.tag == DDOG_PROF_PROFILE_ADD_RESULT_ERR) {
-        std::string ddog_errmsg(ddog_Error_message(&address.err).ptr);
+    ddog_prof_Profile_Result res = ddog_prof_Profile_add(ddog_profile, sample, 0);
+    if (res.tag == DDOG_PROF_PROFILE_RESULT_ERR) {
+        std::string ddog_errmsg(ddog_Error_message(&res.err).ptr);
```

Comment:
is this string then used?

Reply:
Missed that.  No, that was pointless on my part.  I have it printing the string as per the convention used elsewhere in the file, but as we discussed this will be changed to a push-to-logger pattern in a future PR.